cockroachcloud/cmek-ops-gcp.md

```diff
@@ -5,7 +5,7 @@ toc: true
 docs_area: manage.security
 ---
 
-This page covers the procedures required to provision a Customer Managed Encryption Key (CMEK) for your {{ site.data.products.dedicated }} cluster with Google Cloud Platform (GCP).
+This page covers the procedures required to provision [Customer-Managed Encryption Keys (CMEK)](cmek.html) for your {{ site.data.products.dedicated }} cluster with Google Cloud Platform (GCP).
```

Comment:
The difference in singular / plural between the intro sentence for AWS and GCP doesn't seem intentional. Does it make sense to make them as parallel as possible?